meson.build

```diff
@@ -21,12 +21,11 @@ project(
 version = meson.project_version()
 python3 = find_program('python3')
```

Comment:
As Python isn't used anymore here the call to `find_program('python3')` could be moved in the `if get_option('cpp-httplib_compile')` section